src/main/java/org/folio/service/orders/flows/update/open/OpenCompositeOrderPieceService.java

```diff
@@ -168,10 +164,8 @@ public Future<Void> updatePieceRecord(Piece piece, RequestContext requestContext
   public Future<Void> openOrderUpdateInventory(CompositePoLine compPOL, Piece piece, boolean isInstanceMatchingDisabled, RequestContext requestContext) {
     if (Boolean.TRUE.equals(compPOL.getIsPackage())) {
       return titlesService.getTitleById(piece.getTitleId(), requestContext)
-        .compose(title -> inventoryInstanceManager.openOrderHandlePackageLineInstance(title, isInstanceMatchingDisabled, requestContext))
```

Comment:
Just realized that this method openOrderUpdateInventory() is used for non-package POLs. We don't need any changes in flow for non-package orders, so please revert changes in this class.

When adding title to Package POL TitleService is invoked so we need change only in this service for package orders

Reply:
if we revert this change shadow instance can not be used in this place and new instance will be created that is not desired behaviour.
So you fixed here issue when createShadowInstanceIfNeeded not invoked